xero-app-store.yml

```diff
@@ -0,0 +1,260 @@
+                                    "name": "Small",
+                                    "type": "FIXED"
+                                  },
+                                  "startDate": "2021-07-20T03:13:48Z"
+                                }
+                              ]
+                            }
+                          ],
+                          "startDate": "2021-07-20T03:13:48Z",
+                          "status": "ACTIVE"
+                        }
+                      '
+        "404":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+          description: "When a failure occurs in the endpoint"
+      security:
+        - OAuth2: [marketplace.billing]
+components:
+  securitySchemes:
+    OAuth2:
+      type: oauth2
+      description: For more information
+      flows:
+        authorizationCode:
+          authorizationUrl: "https://login.xero.com/identity/connect/authorize"
+          tokenUrl: "https://identity.xero.com/connect/token"
+          scopes:
+            marketplace.billing: Grant read-only access to fixed assets
+  parameters:
+    subscriptionId:
+      required: true
+      in: path
+      name: subscriptionId
+      x-snake: subscription_id
+      description: Unique identifier for Subscription object
+      example: "00000000-0000-0000-0000-000000000000"
+      x-example-java: UUID.fromString("00000000-0000-0000-0000-000000000000")
+      x-example-php: '&quot;00000000-0000-0000-0000-000000000000&quot;'
+      x-example-csharp: Guid.Parse("00000000-0000-0000-0000-000000000000");
+      schema:
+        type: string
+        format: uuid
+  schemas:
+    Subscription:
+      type: object
+      properties:
+        currentPeriodEnd:
+          description: |
+            End of the current period that the subscription has been invoiced for.
+          format: date-time
+          type: string
+        endDate:
+          description: If the subscription has been canceled, this is the date when the subscription ends. If null, the subscription is active and has not been cancelled
+          format: date-time
+          type: string
+        id:
+          description: The unique identifier of the subscription
+          format: uuid
+          type: string
+        organisationId:
+          description: The Xero generated unique identifier for the organisation
+          format: uuid
+          type: string
+        plans:
+          description: List of plans for the subscription.
+          type: array
+          items:
+            $ref: "#/components/schemas/Plan"
+        startDate:
+          description: Date when the subscription was first created.
+          format: date-time
+          type: string
+        status:
+          description: Status of the subscription. Available statuses are ACTIVE, CANCELED, and PAST_DUE.
+          type: string
+        testMode:
```

Comment:
Based on the [docs ](https://developer.xero.com/documentation/api/xero-app-store/subscriptions#get-subscription), 
 this param's name should be "isTestMode".

Reply:
I don't think docs are accurate in this case. Can always update if the API actually changes to reflect that though.